src/matter/cluster/LevelControlCluster.ts

```diff
@@ -0,0 +1,181 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { OptionalAttribute, WritableAttribute, OptionalWritableAttribute, Attribute, Cluster, Command, OptionalCommand, TlvNoResponse } from "./Cluster";
+import { BitFlag, MatterApplicationClusterSpecificationV1_0, TlvBitmap, TlvEnum, TlvField, TlvNullable, TlvObject, TlvUInt16, TlvUInt8 } from "@project-chip/matter.js";
+
+/** @see {@link MatterApplicationClusterSpecificationV1_0} § 1.6.5.8 */
+const optionsBitmap = TlvBitmap(TlvUInt8, {
+    /** Dependency on On/Off cluster. */
+    executeIfOff: BitFlag(0), // TODO default: true.
```

Comment:
where default true is defined?

Reply:
I don't know where I saw that :-) - comment removed